src/handlers/erase_handler.rs

```diff
@@ -74,7 +77,29 @@ pub async fn start(config: Config) -> Result<()> {
 
                 info!("Successfully removed neovim's installation PATH from registry");
             }
+        } else {
+            use what_the_path::shell::Shell;
+            use crate::helpers::directories::get_downloads_directory;
 
+            let shell = Shell::detect_by_shell_var()?;
+
+            match shell {
+                Shell::Fish(fish) => {
+                    let files = fish.get_rcfiles()?;
+                    let fish_file = files[0].join("bob.fish");
+                    if !fish_file.exists() { return Ok(()) }
+                    fs::remove_file(fish_file).await?;
+                },
+                shell => {
+                    let files = shell.get_rcfiles()?;
+                    let downloads_dir = get_downloads_directory(&config).await?;
```

Comment:
I think `downloads_dir` is removed in line 57?

Reply:
you are right i need to fix this, thanks.